src/components/ScreenWrapper/index.js

```diff
@@ -55,6 +56,15 @@ class ScreenWrapper extends React.Component {
             this.setState({didScreenTransitionEnd: true});
             this.props.onEntryTransitionEnd();
         });
+
+        if (this.props.shouldDismissKeyboardBeforeClose) {
```

Comment:
A comment explaining why we doing this might be helpful I would say

Reply:
@mountiny we already have comment in prop types for this field - do we want to add more description?

Reply:
Agreed with Miroslav, basically some context for this change is useful